doc/overview/migrating.rst

```diff
@@ -37,6 +37,8 @@ below which and above which to filter out data.
 +---------------------+----------------------------------------------------------+--------------------------------------------------------------------------------------------------+
 | Filter data         | ``EEG = pop_eegfiltnew(EEG, l_freq, h_freq);``           | :func:`raw.filter(l_freq, h_freq) <mne.io.Raw.filter>`                                           |
 +---------------------+----------------------------------------------------------+--------------------------------------------------------------------------------------------------+
+| Filter data         | ``EEG = pop_eegfiltnew(EEG, l_freq, h_freq);``           | :func:`raw.filter(l_freq, h_freq) <mne.io.Raw.filter>`                                           |
++---------------------+----------------------------------------------------------+--------------------------------------------------------------------------------------------------+
```

Comment:
@apoorva6262 it seems like you forgot to switch branches before working on a new pull request: This was probably meant to be part of a new project, right?